### PR TITLE
Add Stackdriver Debugger/Trace Support

### PR DESCRIPTION
**Pull request by johnsonj**
Based off of changes from @joshlong / @saturnism 

These services require a service account with appropriate permissions. No actual service is created (like Cloud ML). 

Validation:

- Deployed service broker, created instances, bound to an app. Verified the service account in the ENV had appropriate access
- Some light integration testing

### CODE REVIEW

File: brokerapi/brokers/account_managers/service_account_manager.go [hunk 1]
```diff
@@ -111,7 +114,7 @@ func (sam *ServiceAccountManager) CreateAccountInGoogle(instanceID string, bindi
 
 	// create and save key
 	saKeyService := iam.NewProjectsServiceAccountsKeysService(iamService)
-	newSAKey, err := saKeyService.Create(newSA.Name, &iam.CreateServiceAccountKeyRequest{}).Do()
+	newSAKey, err := saKeyService.Create(newSA.Name, &iam.CreateServiceAccountKeyRequest{PrivateKeyType: jsonKeyType}).Do()
```
Comment (ccemeraldeyes) [inline brokerapi/brokers/account_managers/service_account_manager.go]: If there's only one key type do we still need this param?

File: brokerapi/brokers/stackdriver_debugger/broker.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	ProjectId             string
+	Logger                lager.Logger
+	ServiceAccountManager *account_managers.ServiceAccountManager
+	broker_base.BrokerBase
+}
+
+type InstanceInformation struct {
+}
+
+// Creates a service account for Stackdriver Debugger
```
Comment (ccemeraldeyes) [inline brokerapi/brokers/stackdriver_debugger/broker.go]: //does nothing*

File: brokerapi/brokers/stackdriver_debugger/broker.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+func (b *StackdriverDebuggerBroker) Provision(instanceId string, details models.ProvisionDetails, plan models.PlanDetails) (models.ServiceInstanceDetails, error) {
+	return models.ServiceInstanceDetails{}, nil
+}
+
+// Deletes the topic associated with the given instanceID
```
Comment (ccemeraldeyes) [inline brokerapi/brokers/stackdriver_debugger/broker.go]: // also does nothing*

File: brokerapi/brokers/account_managers/service_account_manager.go (hunk 1 above)
Comment (johnsonj) [inline brokerapi/brokers/account_managers/service_account_manager.go]: nahh, removing

File: brokerapi/brokers/stackdriver_trace/broker.go [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+	ProjectId             string
+	Logger                lager.Logger
+	ServiceAccountManager *account_managers.ServiceAccountManager
+	broker_base.BrokerBase
+}
+
+type InstanceInformation struct {
+}
+
+// Creates a service account for Stackdriver Trace
```
Comment (ccemeraldeyes) [inline brokerapi/brokers/stackdriver_trace/broker.go]: does nothing

File: fakes/fake_env_vars.go [hunk 5]
```diff
@@ -105,6 +105,36 @@ const Services string = `[
...
+          "bindable": true,
+          "plan_updateable": false,
+          "metadata": {
+            "displayName": "Google Stackdriver Debugger",
+            "longDescription": "Google Stackdriver Debugger provides powerful production diagnostics tools",
+            "documentationUrl": "https://cloud.google.com/debugger/docs/",
+            "supportUrl": "https://cloud.google.com/support/",
+            "imageUrl": "https://cloud.google.com/_static/images/cloud/products/logos/svg/debugger.svg"
+          },
+          "tags": ["gcp", "debugger"]
```
Comment (ccemeraldeyes) [inline fakes/fake_env_vars.go]: + stackdriver?

File: fakes/fake_env_vars.go [hunk 6]
```diff
@@ -105,6 +105,36 @@ const Services string = `[
...
+          "bindable": true,
+          "plan_updateable": false,
+          "metadata": {
+            "displayName": "Stackdriver Trace",
+            "longDescription": "Stackdriver Trace is a distributed tracing system that collects latency data from your applications and displays it in the Google Cloud Platform Console. You can track how requests propagate through your application and receive detailed near real-time performance insights.",
+            "documentationUrl": "https://cloud.google.com/trace/docs/",
+            "supportUrl": "https://cloud.google.com/support/",
+            "imageUrl": "https://cloud.google.com/_static/images/cloud/products/logos/svg/trace.svg"
+          },
+          "tags": ["gcp", "trace"]
```
Comment (ccemeraldeyes) [inline fakes/fake_env_vars.go]: +stackdriver?

File: brokerapi/brokers/stackdriver_debugger/broker.go (hunk 2 above)
Comment (johnsonj) [inline brokerapi/brokers/stackdriver_debugger/broker.go]: ✔️

File: brokerapi/brokers/stackdriver_debugger/broker.go (hunk 3 above)
Comment (johnsonj) [inline brokerapi/brokers/stackdriver_debugger/broker.go]: ✔️

File: brokerapi/brokers/stackdriver_trace/broker.go (hunk 4 above)
Comment (johnsonj) [inline brokerapi/brokers/stackdriver_trace/broker.go]: ✔️

File: fakes/fake_env_vars.go (hunk 5 above)
Comment (johnsonj) [inline fakes/fake_env_vars.go]: ✔️

File: fakes/fake_env_vars.go (hunk 6 above)
Comment (johnsonj) [inline fakes/fake_env_vars.go]: ✔️

File: README.md [hunk 7]
```diff
@@ -313,6 +313,14 @@ Notes:
         }
         ```
 
+* [Stackdriver Debugger](https://cloud.google.com/debugger/)
+    * Provison (none)
+    * Bind (none)
```
Comment (ccemeraldeyes) [inline README.md]: please add a note saying what the role will be. Ditto for below.

Comment (ccemeraldeyes) [inline README.md]: (and then LGTM!)

Comment (johnsonj) [inline README.md]: :heavy_check_mark:

**Pull request merged**